src/sync/installation.ts

```diff
@@ -516,7 +515,7 @@ const updateRepo = async (subscription: Subscription, repoId: number, values: Re
 };
 
 const getBackfillSince = async (subscription: Subscription, data: BackfillMessagePayload): Promise<Date | null> => {
-	const commitSince = await getCommitSinceDate(data.jiraHost, NumberFlags.SYNC_MAIN_COMMIT_TIME_LIMIT, data.commitsFromDate);
+	const commitSince = data.commitsFromDate ? new Date(data.commitsFromDate) : undefined;
```

Comment:
/nit ternary vs &&... fight!!! I've always leaned towards the `&&` approach, but don't need to change this, just an observation for discussion at some point

`const commitSince = data.commitsFromDate ? new Date(data.commitsFromDate) : undefined;`
`const commitSince = data.commitsFromDate && new Date(data.commitsFromDate);`

Reply:
sorry I want to stick to my origin way of the ternary vs && fight. If `data.commitsFromDate is "" or 0 or NaN`, then `data.commitsFromDate && new Date(data.commitsFromDate)` will evaluated to `"" or 0 or NaN` . However, I want to make sure they evaluated to undefined, hence stick to my origin way. Let me know if you really want me to change it to the &&.